src/runtime/guid.rs

```diff
@@ -131,6 +132,15 @@ impl From<&str> for Guid {
     }
 }
 
+impl Hash for Guid {
```

Comment:
Any reason not to simply derive this?

Reply:
Nope, I forgot you could just do that. I've updated the PR.